log/field.go

```diff
@@ -143,6 +143,16 @@ func Object(key string, obj interface{}) Field {
 	}
 }
 
+// Event adds a string-valued field with name "value" to a Span.LogFields() record
```

Comment:
description is not accurate. Suggest:

`Event creates a string-valued Field for span logs with key="event" and value=val.`

Reply:
Ok, fixed in fresh version. Thank you for your patience.